GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -529,6 +537,25 @@ protected override void OnUICommandsChanged(GitUICommandsChangedEventArgs e)
             base.OnUICommandsChanged(e);
         }
 
+        protected override void WndProc(ref Message m)
+        {
+            if (m.Msg == _closeAllMessage || (m.Msg == NativeMethods.WM_SYSCOMMAND && m.WParam.ToInt32() == NativeMethods.SC_CLOSE))
```

Comment:
`IntPtr.ToInt32` can lead to an overflow exception in some cases. Better to cast to `nint` (if we have it in net6), or cast as `(int)(long)`.

Reply:
Related https://github.com/dotnet/winforms/pull/5791